### PR TITLE
Default to 100.64.0.0/13 as IPv4 service cluster IP range

### PR DESCRIPTION
**Pull request by hakman**
Closes https://github.com/kubernetes/kops/issues/15862

### CODE REVIEW

File: upup/pkg/fi/cloudup/populate_cluster_spec.go [hunk 1]
```diff
@@ -394,14 +394,8 @@ func (c *populateClusterSpec) assignSubnets(cluster *kopsapi.Cluster) error {
...
-			// Allocate from the '0' subnet; but only carve off 1/4 of that (i.e. add 1 + 2 bits to the netmask)
-			serviceOnes := nmOnes + 3
-			// Max size of network is 20 bits
-			if nmBits-serviceOnes > 20 {
-				serviceOnes = nmBits - 20
-			}
-			cidr := net.IPNet{IP: nonMasqueradeCIDR.IP.Mask(nonMasqueradeCIDR.Mask), Mask: net.CIDRMask(serviceOnes, nmBits)}
-			cluster.Spec.Networking.ServiceClusterIPRange = cidr.String()
+			cluster.Spec.Networking.ServiceClusterIPRange = "100.64.0.0/13"
+			cluster.Spec.Networking.ServiceClusterIPRange = "100.64.0.0/13"
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/populate_cluster_spec.go:398]: Line is duplicated

**Comment by adityanmishra** [comment]
This enhancement has broken cluster upgrades from older versions.
We are getting below error now
Unable to create token for CNI kubeconfig error=Post "https://172.21.0.1:443/api/v1/namespaces/kube-system/serviceaccounts/calico-cni-plugin/token": tls: failed to verify certificate: x509: certificate is valid for 100.64.0.1, 127.0.0.1, not 172.21.0.1

**Pull request merged**